doc/vim-lsp.txt

```diff
@@ -2114,6 +2114,14 @@ call `root_uri` function to detect the workspace folder. If the folder is not
 part of workspace folder, it will automatically notify the server to add the
 workspace folder.
 
+==============================================================================
+Show Document                                         *vim-lsp-show-document*
+
+Show Document is an experimental feature of vim-lsp. To enable this, set
+`let g:lsp_experimental_show_document = 1`.
```

Comment:
I would remove `experimental`. I use this only when it requires extensive amount of test and we should probably not enable it. This one seems simple enough that we can enable it and remove experimental.